roles/aws/aws_rds/defaults/main.yml

```diff
@@ -19,3 +19,71 @@ aws_rds:
   master_user_password: hellothere
   publicly_accessible: false # Wether to allocate an IP address
   security_groups: []
+  cloudwatch_alarms:
```

Comment:
Consider calling this `rds_cloudwatch_alarms` for readability later / in debug output / etc.